### ISSUE TITLE
Assigning to ENTRY parsed as error

### ISSUE THREAD

**Issue by gustason** [BalticAmadeus/AblFormatter#231]
For example,
``` abl
typeface = "a,b,c,d,e".
ENTRY(3, typeface, ",") = "xo".
DISPLAY ENTRY(3, typeface, ",") FORMAT "x(16)".
```
![image](https://github.com/user-attachments/assets/3f78bf31-a24d-412e-823a-95425324bdc9)

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/95

**Comment by eglekaz** [comment]
> set-size(mBlobData) = 0.

This was moved to separate issue.

**Comment by eglekaz** [comment]
Fixed with version 0.0.44 of tree-sitter-abl
